### ISSUE TITLE
Name of files

### ISSUE THREAD

**Issue by lionelkusch** [mind-inria/hidimstat#39]
There is a nonhomogenous way to name files and the methods. 

We need to decide if we used a unique acronym for the names of files or/and functions or the full name.

For example, the file adaptive_permutation_threshold.py contains the function ada_svr, and the file loco.py contains loco.

Moreover, the test for knockoffs.py is the only one that has the name of the function test_model_x_knockoff.

**Comment by bthirion** [comment]
I agree that we should have as standard naming a possible.
Sometimes the name have a historical reason (it is the way the methods were branded initially).
That's why it's useful to discuss all name change we propose to do in an issues, e.g. that one.

**Comment by bthirion** [comment]
In general, unsued code should be removed.

**Comment by lionelkusch** [comment]
In the PR #152, @jpaillard proposes to have the name of the file fully written. 
However, in the PR #73, I replaced the name of the file with its short name. 

We need to make a decision on the way to name the file. 
I am more in favour of having the short name of the file, the class and the object because, in other cases, the import of the function will be very long.
What do @bthirion think?

**Comment by bthirion** [comment]
Well 
```
from hidimstat.conditional_permutation_importance import CPI
```
is better than
```
from hidimstat.cpi import CPI
```
because people don't know a priori what CPI means

**Comment by lionelkusch** [comment]
Firstly, I think that people will do `from hidimstat import CPI` more than from `from hidimstat.conditional_permutation_importance import CPI` because the model is accessible directly from the root of the packages.

**Comment by lionelkusch** [comment]
Secondly, for `ada_svr`, following your remarks, I should rename it `adaptative_permutation_threshold_SVR`.
If it's what you want to stay with the full name, I will do a PR to rename this file.

**Comment by bthirion** [comment]
`ada_svr` does not amke sense to anybody  so `adaptative_permutation_threshold_svr` is better anyhow... remember however that all this will probably be dropped in the future.

**Comment by lionelkusch** [comment]
Yes, I know, but as long as the code is in the library, it will be treated like the rest of the code.

**Comment by bthirion** [comment]
So +1 for renaming in the meantime.

**Comment by jpaillard** [comment]
I agree that full names are clearer and we should consider which "flagship" methods we want to put forward and can be directly imported from the root.

**Comment by jpaillard** [comment]
I agree that we should limit acronyms. However,  for CPI and loco, the full name is unreasonably long IMO. 

They do the same for SVC, SVR in sklearn

**Comment by bthirion** [comment]
I think that LOCO is fairly standard.  Less sure about CPI.

**Comment by lionelkusch** [comment]
The goal is to have more explicit names for the names of the classes. However, we can still propose additionally some acronyms for the same methods if it's more useful for people.
